### DISCUSSION TITLE
Kiosk Mode for Share

### DISCUSSION THREAD

**Discussion by pbosh** [oobabooga/text-generation-webui#2968] [Ideas]
I have a need for Kiosk Mode when Share is enabled. I think this would be a powerful feature.

For clarity: I want to share across the internet to people but I don't want them to have access to the settings. Just the Text Generation page. It would allow multiple people to use a model running on my local system at the same time.

**Comment by nsitnov** [comment]
Any suggestions how to do it?

**Reply by missionfloyd** [reply]
#3063
